lib/fluent/log.rb

```diff
@@ -138,6 +138,7 @@ def initialize(logger, opts={})
       @optional_attrs = nil
 
       @suppress_repeated_stacktrace = opts[:suppress_repeated_stacktrace]
+      @forced_stacktrace_level = nil
```

Comment:
It might be `stacktrace_log_level` is enough.

`forced_` prefix gives us the impression that log will be flushed forcibly (or something so special).
As it actually depends on log_level, let's name it simple.